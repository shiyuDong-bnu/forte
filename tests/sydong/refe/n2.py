import psi4
import forte
psi4.set_output_file("n2.out",False)
psi4.set_num_threads(10)
psi4.set_memory("3 gb")
psi4.set_options(
   {
  "basis"            :     "6-31g" ,
  "reference"        :     "rhf",
  "scf_type"         :     "pk",
  "d_convergence"    :     "8",
  "e_convergence"    :     "12",
   }
)
forte_options={
    "ACTIVE_SPACE_SOLVER":"fci",
    "CORRELATION_SOLVER" : "sa-mrdsrg",
    "CORR_LEVEL"         :"ldsrg2",
    "restricted_docc":         [2,0,0,0,0,2,0,0],
    "active":                  [1,0,1,1,0,1,1,1],
    "DSRG_S":1.0,
}
mol=psi4.geometry(
"""
  0 1
  N
  N  1 R
  R = 1.1
"""
)
e,wfn=psi4.energy("scf",return_wfn=True)

e_forte=psi4.energy("forte",forte_options=forte_options,ref_wfn=wfn)
