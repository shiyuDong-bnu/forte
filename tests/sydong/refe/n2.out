
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {} zzzzzzz 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Thursday, 25 April 2024 03:04PM

    Process ID: 9335
    Host:       sydong
    PSIDATADIR: /home/sydong/miniconda3/envs/p4env/share/psi4
    Memory:     500.0 MiB
    Threads:    1
      Threads set to 10 by Python driver.

  Memory set to   2.794 GiB by Python driver.

Scratch directory: /tmp/
   => Libint2 <=

    Primary   basis highest AM E, G, H:  5, 4, 3
    Auxiliary basis highest AM E, G, H:  6, 5, 4
    Onebody   basis highest AM E, G, H:  6, 5, 4
    Solid Harmonics ordering:            Gaussian

*** tstart() called on sydong
*** at Thu Apr 25 15:04:11 2024

   => Loading Basis Set <=

    Name: 6-31G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry N          line   102 file /home/sydong/miniconda3/envs/p4env/share/psi4/basis/6-31g.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                       10 Threads,   2861 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D_inf_h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         N            0.000000000000     0.000000000000    -0.550000000000    14.003074004430
         N            0.000000000000     0.000000000000     0.550000000000    14.003074004430

  Running in d2h symmetry.

  Rotational constants: A = ************  B =      1.98984  C =      1.98984 [cm^-1]
  Rotational constants: A = ************  B =  59653.84489  C =  59653.84489 [MHz]
  Nuclear repulsion =   23.572439384390908

  Charge       = 0
  Multiplicity = 1
  Electrons    = 14
  Nalpha       = 7
  Nbeta        = 7

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: 6-31G
    Blend: 6-31G
    Number of shells: 10
    Number of basis functions: 18
    Number of Cartesian functions: 18
    Spherical Harmonics?: false
    Max angular momentum: 1

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:              10
      Number of primitives:             28
      Number of atomic orbitals:        18
      Number of basis functions:        18

      Integral cutoff                 1.00e-12
      Number of threads:                10

  Performing in-core PK
  Using 29412 doubles for integral storage.
  We computed 4154 shell quartets total.
  Whereas there are 1540 unique shell quartets.
   169.74 percent of shell quartets recomputed by reordering.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:             2145
    Schwarz Cutoff:          1E-12

    OpenMP threads:             10

  Minimum eigenvalue in the overlap matrix is 1.4896792601E-02.
  Reciprocal condition number of the overlap matrix is 5.0517722279E-03.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     Ag         5       5 
     B1g        0       0 
     B2g        2       2 
     B3g        2       2 
     Au         0       0 
     B1u        5       5 
     B2u        2       2 
     B3u        2       2 
   -------------------------
    Total      18      18
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:  -108.05840575690014   -1.08058e+02   0.00000e+00 
   @RHF iter   1:  -108.86039484885903   -8.01989e-01   1.27146e-02 DIIS/ADIIS
   @RHF iter   2:  -108.86720008397234   -6.80524e-03   2.79346e-03 DIIS/ADIIS
   @RHF iter   3:  -108.86760896398700   -4.08880e-04   4.46599e-04 DIIS/ADIIS
   @RHF iter   4:  -108.86761809227099   -9.12828e-06   7.17622e-05 DIIS
   @RHF iter   5:  -108.86761836278858   -2.70518e-07   1.38593e-05 DIIS
   @RHF iter   6:  -108.86761837301351   -1.02249e-08   4.26241e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -15.718623     1B1u  -15.715262     2Ag    -1.525346  
       2B1u   -0.775804     3Ag    -0.629380     1B2u   -0.621235  
       1B3u   -0.621235  

    Virtual:                                                              

       1B3g    0.149789     1B2g    0.149789     3B1u    0.588291  
       4Ag     0.787286     2B2u    0.848179     2B3u    0.848179  
       5Ag     0.944250     2B3g    1.009518     2B2g    1.009518  
       4B1u    1.151183     5B1u    1.567432  

    Final Occupation by Irrep:
             Ag   B1g   B2g   B3g    Au   B1u   B2u   B3u 
    DOCC [     3,    0,    0,    0,    0,    2,    1,    1 ]
    NA   [     3,    0,    0,    0,    0,    2,    1,    1 ]
    NB   [     3,    0,    0,    0,    0,    2,    1,    1 ]

  @RHF Final Energy:  -108.86761837301351

   => Energetics <=

    Nuclear Repulsion Energy =             23.5724393843909077
    One-Electron Energy =                -193.6663440060485186
    Two-Electron Energy =                  61.2262862486440866
    Total Energy =                       -108.8676183730135136

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :          0.0000000            0.0000000            0.0000000
 Dipole Y            :          0.0000000            0.0000000            0.0000000
 Dipole Z            :          0.0000000            0.0000000            0.0000000
 Magnitude           :                                                    0.0000000

 ------------------------------------------------------------------------------------

*** tstop() called on sydong at Thu Apr 25 15:04:11 2024
Module time:
	user time   =       3.07 seconds =       0.05 minutes
	system time =       0.13 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       3.07 seconds =       0.05 minutes
	system time =       0.13 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes

Scratch directory: /tmp/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch: main - git commit: db402211

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

  Forte will use options passed as a dictionary. Option read from psi4 will be ignored


  Preparing forte objects from a Psi4 Wavefunction object
  Read options for space RESTRICTED_DOCC
  Read options for space ACTIVE
  Read options for space RESTRICTED_DOCC

  ==> MO Space Information <==

  -------------------------------------------------------------------------
                       Ag   B1g   B2g   B3g    Au   B1u   B2u   B3u   Sum
  -------------------------------------------------------------------------
    FROZEN_DOCC         0     0     0     0     0     0     0     0     0
    RESTRICTED_DOCC     2     0     0     0     0     2     0     0     4
    GAS1                1     0     1     1     0     1     1     1     6
    GAS2                0     0     0     0     0     0     0     0     0
    GAS3                0     0     0     0     0     0     0     0     0
    GAS4                0     0     0     0     0     0     0     0     0
    GAS5                0     0     0     0     0     0     0     0     0
    GAS6                0     0     0     0     0     0     0     0     0
    RESTRICTED_UOCC     2     0     1     1     0     2     1     1     8
    FROZEN_UOCC         0     0     0     0     0     0     0     0     0
    Total               5     0     2     2     0     5     2     2    18
  -------------------------------------------------------------------------   => Loading Basis Set <=

    Name: 6-31G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry N          line   102 file /home/sydong/miniconda3/envs/p4env/share/psi4/basis/6-31g.gbs 


  Checking orbital orthonormality against current geometry ... Done (OK)

   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: MINAO_BASIS
    atoms 1-2 entry N          line    71 file /home/sydong/miniconda3/envs/p4env/share/psi4/basis/sto-3g.gbs 


  State Singlet (Ms = 0) Ag GAS min: 0 0 0 0 0 0 ; GAS max: 12 0 0 0 0 0 ; weights:
      1.000000000000
  Forte will use psi4 integrals

  ==> Primary Basis Set Summary <==

  Basis Set: 6-31G
    Blend: 6-31G
    Number of shells: 10
    Number of basis functions: 18
    Number of Cartesian functions: 18
    Spherical Harmonics?: false
    Max angular momentum: 1


  JK created using conventional PK integrals
  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:              10
      Number of primitives:             28
      Number of atomic orbitals:        18
      Number of basis functions:        18

      Integral cutoff                 1.00e-12
      Number of threads:                10

  Performing in-core PK
  Using 29412 doubles for integral storage.
  We computed 4154 shell quartets total.
  Whereas there are 1540 unique shell quartets.
   169.74 percent of shell quartets recomputed by reordering.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:             2288
    Schwarz Cutoff:          1E-12

    OpenMP threads:             10



  ==> Integral Transformation <==

  Number of molecular orbitals:                         18
  Number of correlated molecular orbitals:              18
  Number of frozen occupied orbitals:                    0
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                 Conventional


  Computing Conventional Integrals	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Constructing frozen core operators
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.

  Integral transformation done. 0.00956854 s
  Reading the two-electron integrals from disk
  Size of two-electron integrals:   0.002346 GB
  Timing for conventional integral transformation:            0.043 s.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Timing for computing conventional integrals:                0.043 s.

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              3
    number of beta electrons                               3
    number of alpha strings                               20
    number of beta strings                                20
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                                56
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------
  Initial guess space is incomplete.
  Adding 1 determinant(s).

  ==> Initial Guess <==

  Initial guess determinants:         51

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
    27       1       *
    18       3        
     6       5        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0     -108.946978869722  -0.000000  added
  -------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-09
    Residual convergence threshold                 1.000e-06
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                     56
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: adding 1 guess vectors
  Iteration     Average Energy            max(∆E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0     -108.946978869722      108.946978869722        0.009448104945      1
       1     -108.947010137152        0.000031267430        0.001302450231      2
       2     -108.947010683960        0.000000546808        0.000190663659      3
       3     -108.947010701536        0.000000017577        0.000062408020      4
       4     -108.947010703327        0.000000001791        0.000012242260      5
       5     -108.947010703406        0.000000000079        0.000002054865      6
       6     -108.947010703409        0.000000000002        0.000000256903      7
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    2 0 0 0 2 2     -0.96893172
    2 2 0 0 2 0      0.12858737
    2 0 2 0 0 2      0.12858737
    2 b a 0 b a     -0.08391988
    2 a b 0 a b     -0.08391988
    2 b b 0 a a     -0.05623232
    2 a a 0 b b     -0.05623232
    2 2 2 0 0 0     -0.03902994
    0 2 0 0 2 2      0.03563847
    0 0 2 0 2 2      0.03563847
    2 b a 0 a b     -0.02768769
    2 a b 0 b a     -0.02768769
    2 2 0 0 0 2      0.01697986
    2 0 2 0 2 0      0.01697986
    b 0 a a b 2      0.01360460
    b a 0 a 2 b     -0.01360460
    a b 0 b 2 a     -0.01360460
    a 0 b b a 2      0.01360460
    0 0 0 2 2 2      0.01195075

    Total Energy:    -108.947010703409, <S^2>: 0.000000
    Time for FCI:       0.006506617000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)    Ag     0     -108.947010703409   0.000000
    --------------------------------------------------------

  ==> Natural Orbitals <==

        1Ag     1.993244      1B2u    1.940504      1B3u    1.940504  
        1B2g    0.062034      1B3g    0.062034      1B1u    0.001680  


  ==> Dipole Moments [e a0] (Nuclear + Electronic) for Singlet (Ms = 0) Ag <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
         0AG     0.00000000     0.00000000     0.00000000     0.00000000
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     0.00000000     0.00000000
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        1Ag     1.993244      1B2u    1.940504      1B3u    1.940504  
        1B2g    0.062034      1B3g    0.062034      1B1u    0.001680  


  ==> Quadrupole Moments [e a0^2] (Nuclear + Electronic) for Singlet (Ms = 0) Ag <==

       State          QM_XX          QM_XY          QM_XZ          QM_YY          QM_YZ          QM_ZZ
    --------------------------------------------------------------------------------------------------
         0AG    -7.69015334     0.00000000     0.00000000    -7.69015334     0.00000000    -9.48765839
    --------------------------------------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     0.00000000     0.00000000     0.00000000    15.12345955
    --------------------------------------------------------------------------------------------------

  ==> Natural Orbitals <==

        1Ag     1.993244      1B2u    1.940504      1B3u    1.940504  
        1B2g    0.062034      1B3g    0.062034      1B1u    0.001680  


  ==> Semicanonicalize Orbitals <==

    MIX INACTIVE ORBITALS         FALSE
    MIX GAS ACTIVE ORBITALS       FALSE
    FROZEN_DOCC                   CANONICAL
    FROZEN_UOCC                   CANONICAL
    GAS1                          CANONICAL
    RESTRICTED_DOCC               CANONICAL
    RESTRICTED_UOCC               CANONICAL

    Off-Diag. Elements       Max           2-Norm
    ------------------------------------------------
    GAS1                 0.0000000000   0.0000000000
    RESTRICTED_DOCC      0.0001671356   0.0002578741
    RESTRICTED_UOCC      0.0022839703   0.0037870357
    ------------------------------------------------

    Canonicalization test failed

  Integrals are about to be updated.
  Computing Conventional Integrals	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Constructing frozen core operators
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.

  Integral transformation done. 0.00920442 s
  Reading the two-electron integrals from disk
  Size of two-electron integrals:   0.002346 GB
  Timing for conventional integral transformation:            0.045 s.
  Timing for freezing core and virtual orbitals:              0.000 s.
  Integrals update took     0.045 s.
  Timing for orbital canonicalization:                        0.046 s.

    -----------------------------------------------------------------------
      Spin-Adapted Multireference Driven Similarity Renormalization Group
                            written by Chenyang Li
                                10 OMP threads
    -----------------------------------------------------------------------

  Disclaimer:
    The spin-adapted DSRG code is largely adopted from the spin-integrated code developed by
    Chenyang Li, Kevin P. Hannon, Tianyuan Zhang, and Francesco A. Evangelista.

  ==> Multireference Driven Similarity Renormalization Group <==

    Computing Fock matrix and cleaning JK ....... Done. Timing      0.001 s
    Reading DSRG options ........................ Done. Timing      0.000 s
    Setting ambit MO space ...................... Done. Timing      0.000 s
    Initializing density cumulants .............. Done. Timing      0.005 s
    Filling Fock matrix from ForteIntegrals ..... Done. Timing      0.000 s

  ==> Density Cumulant Summary <==

             2-cumulant   3-cumulant
    --------------------------------
    max        0.283075     0.064848
    2-norm     0.873735     0.501709
    --------------------------------

  ==> Checking Semicanonical Orbitals <==

    Block                Max            Mean
    ----------------------------------------
    CORE        0.0000000000    0.0000000000
    VIRTUAL     0.0000000000    0.0000000000
    GAS1        0.0000000000    0.0000000000
    ----------------------------------------
    Orbitals are semi-canonicalized.

  ==> Calculation Information <==

    --------------------------------------------------------
    Correlation level                                 LDSRG2
    Integral type                               CONVENTIONAL
    Source operator                                 STANDARD
    Reference relaxation                                NONE
    3RDM algorithm                                  EXPLICIT
    Core-Virtual source type                          NORMAL
    T1 amplitudes initial guess                          PT2
    Restart amplitudes                                  TRUE
    Sequential DSRG transformation                     FALSE
    Omit blocks of >= 3 virtual indices                FALSE
    Read amplitudes from current dir                   FALSE
    Write amplitudes to current dir                    FALSE
    Flow parameter                                 1.000e+00
    Energy convergence threshold                   1.000e-09
    Residual convergence threshold                 1.000e-06
    Recursive single commutator threshold          1.000e-12
    Taylor expansion threshold                     1.000e-03
    Intruder amplitudes threshold                  1.000e-01
    Max number of iterations                             100
    Max nested commutators                                20
    DIIS start                                             2
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    DIIS extrapolating freq                                1
    Number of amplitudes for printing                     15
    --------------------------------------------------------

  ==> MR-DSRG (LDSRG2) Memory Information <==

    Memory assigned by the user                      3.00 GB
    Memory available for MR-DSRG                     2.70 GB
    Generalized Fock matrix                          2.74 KB
    1-, 2-, and 3-density cumulants                384.19 KB
    1- and 2-electron integrals                    842.40 KB
    T1 cluster amplitudes and residuals              2.24 KB
    T2 cluster amplitudes and residuals            470.40 KB
    1-body Hbar and intermediates                    7.78 KB
    2-body Hbar and intermediates                    2.52 MB
    Local intermediates for commutators            725.76 KB
    Max memory for local intermediates             725.76 KB
    Memory currently available                       2.69 GB

  =>** Before self.dsrg_solver.compute_energy() **<=


  ==> Build Initial Amplitudes Guesses <==

    Computing T2 amplitudes from PT2 ............ Done. Timing      0.002 s
    Computing T1 amplitudes from PT2 ............ Done. Timing      0.001 s

  ==> Initial Excitation Amplitudes Summary <==

    Active Indices:    2    5    7   11   14   16
    Largest T1 amplitudes (absolute values):
        i    a                i    a                i    a
    -----------------------------------------------------------------
    [   7    8] 0.0023183 [   5    6] 0.0023183 [   1    2] 0.0016095
    [  10   11] 0.0014211 [  11   13] 0.0013704 [  11   12] 0.0009915
    [   2    3] 0.0009471 [   1    3] 0.0008354 [  10   12] 0.0006544
    [   1    4] 0.0005128 [  10   13] 0.0004770 [   2    4] 0.0002256
    [  14   15] 0.0001866 [  16   17] 0.0001866 [   9   12] 0.0000502
    -----------------------------------------------------------------
    2-Norm of T1 vector:                            0.004567192338892
    Number of nonzero elements:                                    20
    -----------------------------------------------------------------
    Largest T2 amplitudes (absolute values):
        i    j    a    b                i    j    a    b                i    j    a    b
    -----------------------------------------------------------------------------------------------
    [  10   16    2    5] 0.0868644 [  10   14    2    7] 0.0868644 [  16    5   16    6] 0.0808486
    [  14    7   14    8] 0.0808486 [  14    5   14    6] 0.0736227 [  16    7   16    8] 0.0736227
    [  10    5    2   16] 0.0711857 [  10    7    2   14] 0.0711857 [   2    5    2    6] 0.0703524
    [   2    7    2    8] 0.0703524 [   5    5    5    6] 0.0665410 [   7    7    7    8] 0.0665410
    [  16   16   16   17] 0.0635741 [  14   14   14   15] 0.0635741 [   2   16    2   17] 0.0628074
    -----------------------------------------------------------------------------------------------
    2-Norm of T2 vector:                                                          0.743817949139433
    Number of nonzero elements:                                                                2608
    -----------------------------------------------------------------------------------------------

  ==> Possible Intruders <==

    T1 amplitudes larger than 0.1000: NULL
    T2 amplitudes larger than 0.1000: NULL

  ==> Computing MR-LDSRG(2) Energy <==

    Reference:
      J. Chem. Phys. 2016, 144, 164114.

                  Energy (a.u.)           Non-Diagonal Norm        Amplitude RMS         Timings (s)
           ---------------------------  ---------------------  ---------------------  -----------------
    Iter.        Corr.         Delta       Hbar1      Hbar2        T1         T2        Hbar     Amp.    DIIS
    ---------------------------------------------------------------------------------------------------------
       1    -0.146010629333 -1.460e-01   8.038e-02  5.303e-01   3.201e-02  2.048e-01     2.543    0.004
       2    -0.153413719448 -7.403e-03   2.207e-02  6.171e-01   5.516e-03  4.406e-02     2.416    0.004  S
       3    -0.153859252022 -4.455e-04   3.032e-02  5.784e-01   2.222e-03  1.110e-02     2.449    0.004  S
       4    -0.153881511275 -2.226e-05   2.738e-02  5.825e-01   3.401e-04  3.727e-03     2.516    0.004  S/E
       5    -0.153867756858  1.375e-05   2.754e-02  5.811e-01   5.630e-05  6.513e-04     2.461    0.004  S/E
       6    -0.153865739256  2.018e-06   2.759e-02  5.810e-01   1.721e-05  1.772e-04     2.504    0.004  S/E
       7    -0.153866720046 -9.808e-07   2.758e-02  5.810e-01   5.671e-06  4.372e-05     2.472    0.005  S/E
       8    -0.153866772667 -5.262e-08   2.758e-02  5.810e-01   1.176e-06  1.030e-05     2.428    0.004  S/E
       9    -0.153866626676  1.460e-07   2.758e-02  5.810e-01   5.201e-07  2.220e-06     2.519    0.004  S/E
      10    -0.153866596796  2.988e-08   2.758e-02  5.810e-01   9.227e-08  5.518e-07     2.554    0.004  S/E
      11    -0.153866593465  3.331e-09   2.758e-02  5.810e-01   1.705e-08  1.347e-07     2.435    0.003  S/E
      12    -0.153866593522 -5.754e-11   2.758e-02  5.810e-01   5.208e-09  3.102e-08     2.521    0.004  S/E
    ---------------------------------------------------------------------------------------------------------

  ==> MR-LDSRG(2) Energy Summary <==

    E0 (reference)                 =    -108.947010703408807
    MR-LDSRG(2) correlation energy =      -0.153866593522332
    MR-LDSRG(2) total energy       =    -109.100877296931145

  ==> Final Excitation Amplitudes Summary <==

    Active Indices:    2    5    7   11   14   16
    Largest T1 amplitudes (absolute values):
        i    a                i    a                i    a
    -----------------------------------------------------------------
    [   2    4] 0.0157048 [   7    8] 0.0146533 [   5    6] 0.0146533
    [   1    2] 0.0110757 [  11   13] 0.0105235 [  10   12] 0.0099809
    [  14   15] 0.0094443 [  16   17] 0.0094443 [  10   13] 0.0043474
    [  10   11] 0.0032816 [   2    3] 0.0032552 [   1    4] 0.0019976
    [  11   12] 0.0018025 [   0    2] 0.0002825 [   0    4] 0.0002412
    -----------------------------------------------------------------
    2-Norm of T1 vector:                            0.035146199079566
    Number of nonzero elements:                                    20
    -----------------------------------------------------------------
    Largest T2 amplitudes (absolute values):
        i    j    a    b                i    j    a    b                i    j    a    b
    -----------------------------------------------------------------------------------------------
    [  10   14    2    7] 0.0679911 [  10   16    2    5] 0.0679911 [   2    2    2    4] 0.0655601
    [  10   10    7    7] 0.0617664 [  10   10    5    5] 0.0617664 [  10    2    2   12] 0.0600560
    [  16    5   16    6] 0.0578074 [  14    7   14    8] 0.0578074 [   2   14    2   15] 0.0563214
    [   2   16    2   17] 0.0563214 [   5   16    5   17] 0.0549343 [   7   14    7   15] 0.0549343
    [  10    7    2   14] 0.0547595 [  10    5    2   16] 0.0547595 [  16   16   16   17] 0.0540395
    -----------------------------------------------------------------------------------------------
    2-Norm of T2 vector:                                                          0.672236918417995
    Number of nonzero elements:                                                                2608
    -----------------------------------------------------------------------------------------------

  ==> Possible Intruders <==

    T1 amplitudes larger than 0.1000: NULL
    T2 amplitudes larger than 0.1000: NULL
  =>** After self.dsrg_solver.compute_energy() **<=

  Semicanonical orbitals must be used!


  ==> Total Timings (s) for Computing Commutators <==

           [H1, T1]    [H1, T2]    [H2, T1]    [H2, T2]  
    -----------------------------------------------------
    -> C0       0.042       0.054       0.064       1.168
    -> C1       0.080       0.414       0.551       6.313
    -> C2                   1.579       3.623      15.351
    -----------------------------------------------------


  Time to prepare integrals:        0.140 seconds
  Time to run job          :       30.056 seconds
  Total                    :       30.197 seconds
